google/oauth2/service_account_async.py

```diff
@@ -0,0 +1,606 @@
+# Copyright 2016 Google LLC
```

Comment:
Again, what's the necessity of duplicating this file?

Reply:
The reason we would require an async file is inheritance from the async credentials, but in the latest commit I resolved the code duplication by increasing inheritance in the async methods from the sync methods.